pkg/util/overridemanager/overridemanager.go

```diff
@@ -352,6 +405,50 @@ func applyArgsOverriders(rawObj *unstructured.Unstructured, argsOverriders []pol
 	return nil
 }
 
+func applyFieldOverriders(rawObj *unstructured.Unstructured, FieldOverriders []policyv1alpha1.FieldOverrider) error {
+	if len(FieldOverriders) == 0 {
+		return nil
+	}
+	for index := range FieldOverriders {
+		pointer, err := jsonpointer.New(FieldOverriders[index].FieldPath)
+		if err != nil {
+			klog.Errorf("Build jsonpointer with overrider's path err: %v", err)
+			return err
+		}
+		res, kind, err := pointer.Get(rawObj.Object)
+		if err != nil {
+			klog.Errorf("Get value by overrider's path err: %v", err)
+			return err
+		}
+		if kind != reflect.String {
+			errMsg := fmt.Sprintf("Get object's value by overrider's path(%s) is not string", FieldOverriders[index].FieldPath)
+			klog.Errorf(errMsg)
+			return fmt.Errorf(errMsg)
+		}
+		dataBytes := []byte(res.(string))
+		klog.V(4).Infof("Parsed JSON patches by FieldOverriders[%d](%+v)", index, FieldOverriders[index])
+		var appliedRawData []byte
+		if len(FieldOverriders[index].YAML) > 0 {
```

Comment:
We can only choose `YAML` or `JSON`, so we need to pre-validate that in `pkg/webhook/overridepolicy/validating` as well.